chef/cookbooks/network/recipes/default.rb

```diff
@@ -448,6 +448,12 @@ def kill_nic(nic)
     our_iface = br
     net_ifs << our_iface.name
   end
+
+  if network.ovs_forward_bpdu
```

Comment:
this looks wrong to me. you don't want to see if it is "true" but if it is "defined". because setting it to false should also be done (even if its the code default, it might not have been the setting in the currently installed configuration)

Reply:
oh, good point, thanks